content/SCALE/GettingStarted/FirstTimeLogin.md

```diff
@@ -46,12 +46,46 @@ On a computer with access to the same network as the TrueNAS system, enter the h
 The quality of your user experience can be impacted by the browser that you use. We generally recommend using Firefox, Edge, or Chrome.
 {{< /hint >}}
 
-![LoginSCALE](/images/SCALE/LoginSCALE.png "TrueNAS SCALE Login Screen")
+With the implementation of rootless login and based on your Authentication Method selection in the SCALE [TrueNAS installer]({{< relref "InstallingScale.md" >}}), you could see one of three sign-in options for the web UI:
+
+!![SCALEInstallerConsoleSetupAdminAccount](/images/SCALE/22.12/SCALEInstallerConsoleSetupAdminAccount.png "Admin User Screen")
+
+* 1. Logging in as the new admin user, entering the admin username and password created during installation.
+* 2. Logging in as the root user, entering root as the username and the root password created during installation.
+* 3. Configuring a login when you first log into the UI.
 
 Use the administrative account credentials to log in.
-The default administrator username is `root` and the password is created when installing TrueNAS.
+The default administrator username is no longer root but the administrative user creating during installation.
+
+{{< include file="/_includes/RootLoginDeprecatedSCALE.md" type="page" >}}
+
+### Logging In As Admin
+
+If you set up the admin user during the installation, enter the admin username and password you set up. 
+
+![LoginSCALE](/images/SCALE/LoginSCALE.png "TrueNAS SCALE Login Screen")
+
+Note, the root user still exists but the password is disabled by default so that only the admin user can log into the system. 
+To modify user credentials, go to **Credentials > Local User**, click anywhere on the user row and click **Edit**. For more information see [Managing Users]({{< relref "ManageLocalUsersScale.md" >}}).
```

Comment:
```suggestion
Note, the root user still exists but the password is disabled by default so that only the admin user can log into the system. 
To modify user credentials, go to **Credentials > Local User**, click anywhere on the user row, then click **Edit**. For more information see [Managing Users]({{< relref "ManageLocalUsersScale.md" >}}).
```